sdk/search/Microsoft.Azure.Search/tests/Tests/SkillsetsTests.cs

```diff
@@ -63,18 +84,6 @@ public void CreateSkillsetReturnsCorrectDefinitionOcrHandwritingSentiment()
             });
         }
 
-        [Fact]
-        public void CreateSkillsetThrowsExceptionWithInvalidLanguageSelection()
```

Comment:
Why is this test being removed?

Reply:
A change in the backend makes this test unnecessary, so I asked Arvind to remove it as part of this change (it currently fails if you run it actually).  We can do it in a separate PR if you really want.

Reply:
Also, there is a separate task on me to obsolete-ize the `textExtractionAlgorithm` as it's no longer being used (and hence making the test fail as it's not validated in the service)

Reply:
> A change in the backend makes this test unnecessary, so I asked Arvind to remove it as part of this change (it currently fails if you run it actually). We can do it in a separate PR if you really want.

nit: The test fails if you run it in -Record mode

Reply:
> I have a concern about an apparently unrelated test change.

@Careyjmac and I have added extra context in the thread. Let me know what you think